### ISSUE TITLE
🐛 [Bug] Tank position in raid and group

### ISSUE THREAD

**Issue by xBarselonx** [vmangos/core#1793]
## 🐛 Bug report
 
Hi!
I create bots with .partybot command (tank, dps and healer). When a character moves in a group with bots, the tank sticks to the position behind the group. As it is now I have depicted in the figure below.
![image](https://user-images.githubusercontent.com/99982508/216557409-49472d45-a320-49f0-b814-85f72c348057.jpeg)

### Expected behavior

Since the tank is the role of defense and continuous provocation, I believe that his position should be ahead of the group in front. The tank should be the first to strike the boss or mob. As I suggest in the picture below.

![image](https://user-images.githubusercontent.com/99982508/216557550-265c1178-91ec-4a99-91f5-a40d3f0618f8.jpeg)

### Steps to reproduce

1.
2.
3.
4. ...
5. Profit

### Version & Environment

 
Client Version: 1.12.1

Commit Hash:  

OS Client: Win 10
OS Server: 

### Crashlog

- None

**Comment by mserajnik** [comment]
Just leaving this YouTube channel that I've stumbled upon here (this seemed like the most fitting, partybot-related issue to add this to): https://youtube.com/@otherjelly

As you can see from the numerous videos, this guy is working on improving VMaNGOS partybot behavior, especially in raids.

The channel description says the code is currently not yet published on GitHub, but once/if it does become public, it might be worth checking out and potentially cherry-picking at least some features/enhancements (if the developer doesn't open a PR anyway).

**Comment by gajet5** [comment]
Use tags. on my server, the guys easily go through all the raids using bots. But my bots are not exactly the ones that are in the source. However, I did not change the logic of behavior, I worked more on effects, chemistry, and clothes.

**Comment by mserajnik** [comment]
> on my server, the guys easily go through all the raids using bots.

Quite impressive!

> But my bots are not exactly the ones that are in the source. However, I did not change the logic of behavior, I worked more on effects, chemistry, and clothes.

Do I misremember that you started implementing scripts for the bots for specific fights? 🤔

**Comment by gajet5** [comment]
> Do I misremember that you started implementing scripts for the bots for specific fights? 🤔

No, I don't think it's necessary to put 100% of the work on bots. They should not replace living people.
I have completed sets of clothes with my own, depending on the itl of the leader. I've prescribed behavior for some bosses, and the Hunts take down Frenzy as an example.

Several new commands and improved the old ones.

For example, personal control, if there are 2 people in a group and they have called bots, then everyone controls their own.
